XPlat.ApplicationModel/XPlat.ApplicationModel.csproj

```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'uap10.0' ">
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.1.7" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.1.9" PrivateAssets="All" />
```

Comment:
Is there a reason that we aren't using the 6.2.3 update like MADE?

Reply:
6.2.3 isn't available anymore and was pulled recently I believe. 6.1.9 is the latest although this shouldn't matter, it's a dev dependency.

Reply:
Cool no worries, will we need to make this change on MADE?

Reply:
Possibly. I can take a look at that.